shell/platform/embedder/tests/embedder_unittests.cc

```diff
@@ -1022,7 +1090,8 @@ TEST_F(EmbedderTest, NoLayerCreatedForNoOverlayOnTopOfPlatformLayer) {
   auto scene_image = context.GetNextSceneImage();
 
   context.GetCompositor().SetNextPresentCallback(
-      [&](const FlutterLayer** layers, size_t layers_count) {
+      [&](FlutterViewId view_id, const FlutterLayer** layers,
+          size_t layers_count) {
         ASSERT_EQ(layers_count, 2u);
```

Comment:
Although this value is only passed in by `EmbedderConfigBuilder`, so it shouldn't matter... but that makes the test even more weird. Why is this callback set in such a way, instead of using public API?

Reply:
> Should we add `ASSERT_EQ(view_id, kImplicitViewId);` to all tests like this?

For now, only `EmbedderTest.CanRenderImplicitView` and `EmbedderTest.CanRenderImplicitViewUsingPresentLayersCallback` care about the `view_id`.

The other `EmbedderTest`s only care that the correct content was rendered, they don't care which view it was rendered to. I don't think verifying the view ID is necessary for these tests.

Once we allow multiple views, we will definitely want new tests that verify the view IDs are expected. I expect these to be new tests that are separate from the existing tests.

> Why is this callback set in such a way, instead of using public API?

The tests are doing a bunch of setup magic so that they can render stuff using the different backends.